style.css

```diff
@@ -0,0 +1,783 @@
+}
+
+.humburger.active .bar:nth-child(1) {
+  transform: translateY(8px) rotate(45deg);
+  background-color: #36454f;
+}
+
+.humburger.active .bar:nth-child(3) {
+  transform: translateY(-10px) rotate(-45deg);
+  background-color: #36454f;
+}
+
+.Desk_nav {
+  display: none;
+}
+
+.Desk_nav.active {
+  position: fixed;
+  left: 0;
+  top: 0;
+  flex-direction: column;
+  width: 80%;
+  height: 70vh;
+  text-align: center;
+  transition: 0.3s;
+  padding: 30px 0 0 30px;
+  display: block;
+  backdrop-filter: blur(2px);
+  -webkit-backdrop-filter: blur(2px);
+  background: #272a31;
+}
+
+.Desk_nav.active .Desk_list {
+  font-weight: 600;
+  font-size: 32px;
+  line-height: 44px;
+  display: flex;
+  align-items: center;
+  margin-top: 40px;
+  margin-right: 20px;
+  border: #36454f 1px solid;
+  justify-content: center;
+}
+
+.Desk_nav.active .Desk_list a {
+  color: #fff;
+}
+
+.activity img {
+  display: flex;
+  margin: 0 auto;
+  width: 100px;
+  height: 100px;
+  padding-right: 5px;
+}
+
+.humburger img {
+  width: 30px;
+}
+
+.footerLower img,
+.footerLower1 img {
+  width: 200px;
+}
+
+.footerUpper img,
+.footerUpper1 img {
+  width: 60px;
+}
+
+.Logo img {
+  width: 300px;
+  border: 1px solid #d3d3d3;
+  margin: 20px auto;
+}
+
+.past-events {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  margin-top: 15px;
+  text-align: center;
+}
+
+.past-events hr {
+  align-self: center;
+  width: 40px;
+  color: #ec5242;
+  margin: 15px 0;
+}
+
+.past-events div img {
+  width: 90%;
+  margin: 10px 0 5px 20px;
+  height: 250px;
+  opacity: 0.5;
+  transition: all 0.5s ease-in-out;
+}
+
+.footerUpper ul li img,
+.footerUpper1 ul li img {
+  width: 90px;
+  height: 90px;
+}
+
+.past-events div img:hover {
+  width: 90%;
+  margin: 10px 0 5px 20px;
+  height: 250px;
+  opacity: 1;
+  border: #36454f 1px solid;
+}
+
+.about {
+  display: flex;
+  flex-direction: column;
+}
+
+.Logo {
+  width: 90%;
+  display: flex;
+  flex-direction: column;
+  text-align: center;
+  margin: 0 auto;
+}
+
+.Logo h2 {
+  font-size: 30px;
+  margin: 30px 0 10px 0;
+}
+
+.Logo hr {
+  width: 40px;
+  color: #ec5242;
+  align-self: center;
+  margin: 5px 0 10px 0;
+}
+
+.about h1 {
+  font-size: 37px;
+  color: #ec5242;
+  font-weight: 900;
+  padding: 15px 0;
+}
+
+.about-para {
+  background: white;
+  border: #36454f;
+  padding: 5px;
+  margin-top: 10px;
+}
+
+.about-para1 {
+  padding-top: 20px;
+  font-size: 25px;
+  color: #ec5242;
+}
+
+.about-para2 {
+  margin: 20px 0;
+  width: 90%;
+  align-items: center;
+  font-size: medium;
+}
+
+.introContainer h2 {
+  padding: 15px 0;
+  font-size: 23px;
+  color: #ec5242;
+}
+
+.introContainer h1 {
+  font-size: 37px;
+  color: #ec5242;
+  font-weight: 900;
+  padding: 15px 0;
+}
+
+.para3 {
+  width: 70%;
+  font-size: medium;
+  line-height: 1.5;
+  border: #f4f4f4 3px solid;
+  margin: 10px 0;
+  background-color: white;
+  padding: 5px;
+  text-align: left;
+}
```

Comment:
- [ ] Please notice how this paragraph is centered and taking almost the full width of the mobile screen in the original design guidelines :point_down: 
![image](https://user-images.githubusercontent.com/97521557/217835097-535e0d97-ec21-4cd0-b184-abdfe1f3dd8e.png)

what we have currently :point_down: 
![image](https://user-images.githubusercontent.com/97521557/217835187-9b37979f-7af0-4262-a5bf-ad6b022e5c4f.png)

please fix the width here :+1:

Reply:
I have just realized that and I will make the necessary corrections